optaweb-vehicle-routing-docs/src/main/asciidoc/quickstart.adoc

```diff
@@ -0,0 +1,175 @@
+
+.Procedure
+. Open http://download.geofabrik.de/ in a web browser.
+. Click a region in the *Sub Region* list, for example *Europe*.
+The sub region's page opens.
+. In the *Sub Regions* table, download the OSM file (`.osm.pbf`) for a country, for example Belgium.
+
+=== Create the OptaWeb Vehicle Routing data directory structure
+
+OptaWeb Vehicle Routing reads and writes several types of data on the file system.
+It reads OSM (OpenStreetMap) files from the `openstreetmap` directory, writes a road network graph to the `graphhopper` directory, and persists user data in a directory called `db`.
+Create a new directory dedicated to storing all of these data to make it easier to upgrade to a newer version of OptaWeb Vehicle Routing in the future and continue working with the data you created previously.
+
+.Procedure
+. Create the `openstreetmap` directory in your user account `home` directory, for example:
++
+[source]
+----
+$HOME/.vrp
+└── openstreetmap
+----
+
+. Move all of your downloaded OSM files (files with the extension `.osm.pbf`) to the `openstreetmap` directory.
+
+The rest of the directory structure will be created by the OptaWeb Vehicle Routing application when it runs for the first time.
+After that, your directory structure will look similar to the following example:
+
+// TODO maybe replace this with a screenshot, doesn't look good in PDF.
+[source]
+----
+$HOME/.vrp
+├── db
+│   └── vrp.mv.db
+├── graphhopper
+│   └── belgium-latest
+└── openstreetmap
+    └── belgium-latest.osm.pbf
+----
+
+=== Verify the Java version
+
+// TODO Java 11
+Java SE 8 or higher must be installed on your system before you can use OptaWeb Vehicle Routing.
+It is recommended that you install JDK.
+However, if you have a binary distribution of OptaWeb Vehicle Routing, you only need the Java SE Runtime Environment (JRE).
+But if you want to build OptaWeb Vehicle Routing from source, your system must have the Java SE Development Kit (JDK).
+
+.Procedure:
+. To verify the current Java installation, enter the following command:
++
+[source,shell]
+----
+java -version
+----
+. If necessary, install OpenJDK 8.
+* To install OpenJDK 8 on Fedora, enter the following command:
++
+[source,shell]
+----
+sudo dnf install java-1.8.0-openjdk-devel
+----
+* To install OpenJDK on other platforms, follow instructions at https://openjdk.java.net/install/.
+
+=== Download OptaWeb Vehicle Routing
+
+Download the OptaWeb Vehicle Routing distribution archive, available from the OptaPlanner website, to quickly evaluate OptaWeb Vehicle Routing without having to set up build tools.
+
+.Prerequisites
+* Java 8 or higher is installed.
+
+// TODO cross-reference
+NOTE: If you want to modify OptaWeb Vehicle Routing and build it yourself or contribute to upstream, see the Development chapter.
+
+.Procedure
+. Go to https://www.optaplanner.org/download/download.html
```

Comment:
`https://www.optaplanner.org/download/download.html`

Reply:
What's the purpose of making the link not clickable?